Docs: Update the doc on execution devices property and enable_startup_fallback property (#14750)
openvinotoolkit/openvino: docs/OV_Runtime_UG/auto_device_selection.md

@sphinxdirective

+---------------------------------------------+----------------------------------------------------------------------+
| | Property                                  | | Values and Description                                             |

> +======================================v======+======================================================================+

Reviewer: ```suggestion
+=============================================+======================================================================+
```

| |                                           | | executed.                                                          |
| |                                           | | Examples of returning results could be `(CPU)`(`(CPU)` is a        |
| |                                           | | temporary device, indicating that CPU is used for acceleration at  |
| |                                           | | the model compilation stage), `CPU`, `GPU`, `CPU GPU`, `GPU.0`,    |
| |                                           | | etc.                                                               |
+---------------------------------------------+----------------------------------------------------------------------+
| | `ov::intel_auto::enable_startup_fallback` | |  **Values**:                                                       |
| |                                           | |       `true`                                                       |
| |                                           | |       `false`                                                      |
| |                                           | |                                                                    |

> | |                                           | | Enable/disable CPU as acceleration (or helper device) at the       |

Reviewer: ```suggestion
| |                                           | | Enables/disables CPU as acceleration (or the helper device) in the       |
```

> | |                                           | | beginning. The default value is `true`.                            |

Reviewer: ```suggestion
| |                                           | | beginning. The default value is `true`, indicating that CPU is used as acceleration by default.                            |
```
